hikaricp/tests/fixtures/metrics-prometheus.txt

```diff
@@ -0,0 +1,57 @@
+# HELP hikaricp_connection_timeout_total Connection timeout total count
+# TYPE hikaricp_connection_timeout_total counter
+hikaricp_connection_timeout_total{pool="HikariPool-1",} 0.0
+
+
+
```

Comment:
Is there a specific reason the fixture here has a few empty lines?

Reply:
No reason. I must have missed this when tidying up my code 🙈.
I'll remove

Reply:
This change has been made. Thanks for spotting 👍